runtime/runtime-core/jvm/src/aws/smithy/kotlin/runtime/time/InstantJVM.kt

```diff
@@ -125,6 +125,16 @@ public actual class Instant(internal val value: jtInstant) : Comparable<Instant>
          * Create an [Instant] from the current system time
          */
         public actual fun now(): Instant = Instant(jtInstant.now())
+
+        /**
+         * Create an [Instant] with the minimum possible value
+         */
+        public val MIN: Instant = Instant(jtInstant.MIN)
+
+        /**
+         * Create an [Instant] with the maximum possible value
+         */
+        public val MAX: Instant = Instant(jtInstant.MAX)
```

Comment:
**Correctness**: I believe these should be `actual`s with corresponding `expect`s in **Instant.kt**.

Reply:
Yeah you're right, I completely missed the `actual` and `expects`, I'll update it. And yeah, they should have `_VALUE` for consistency. I think .MAX and .MIN look nicer but we should follow the standard set by Kotlin and Java.